src/__init__.py

```diff
@@ -1 +0,0 @@
-from . import models  # noqa
```

Comment:
**question (non-blocking):** Is this actually why alembic stopped picking up all of my db items?

Reply:
Kinda, but new solution seems best